### ISSUE TITLE
macOS Catalina - crashes on start

### ISSUE THREAD

**Issue by shoguevara** [sidneys/pb-for-desktop#113]
Crashes on start - see attached screenshot

```
Process:               PB for Desktop [12016]
Path:                  /Applications/PB for Desktop.app/Contents/MacOS/PB for Desktop
Identifier:            de.sidneys.pb-for-desktop
Version:               10.2.0 (10.2.0)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           PB for Desktop [12016]
User ID:               501

Date/Time:             2019-11-27 00:27:52.573 +0300
OS Version:            Mac OS X 10.15.1 (19B88)
Report Version:        12
Anonymous UUID:        228D603C-9BEC-41B5-B02F-8E464C94BA1B

Sleep/Wake UUID:       479BDF83-C601-4F3B-9513-352076CEA5A2

Time Awake Since Boot: 40000 seconds
Time Since Wake:       3500 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @rpath/Electron Framework.framework/Electron Framework
  Referenced from: /Applications/PB for Desktop.app/Contents/MacOS/PB for Desktop
  Reason: no suitable image found.  Did find:
	/Applications/PB for Desktop.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework: file too short
	/Applications/PB for Desktop.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework: stat() failed with errno=1

Binary Images:
       0x106c95000 -        0x106cbdfff +de.sidneys.pb-for-desktop (10.2.0 - 10.2.0) <CD315A0C-AE44-3040-8C05-F56E5800F953> /Applications/PB for Desktop.app/Contents/MacOS/PB for Desktop
       0x108fd4000 -        0x109064b5f  dyld (733.6) <DAFEA246-2F9A-3DCB-A37C-4246D4F92770> /usr/lib/dyld
    0x7fff3eb7e000 -     0x7fff3ec35ff9  com.apple.MediaPlayer (1.0 - 1.0) <41C84531-7B33-3169-8CC7-AA4D60B65A1D> /System/Library/Frameworks/MediaPlayer.framework/Versions/A/MediaPlayer

Model: MacBookPro11,1, BootROM 157.0.0.0.0, 2 processors, Dual-Core Intel Core i7, 2,8 GHz, 16 GB, SMC 2.16f68
Graphics: kHW_IntelIrisItem, Intel Iris, spdisplays_builtin
Memory Module: BANK 0/DIMM0, 8 GB, DDR3, 1600 MHz, 0x80AD, 0x484D54343147533641465238412D50422020
Memory Module: BANK 1/DIMM0, 8 GB, DDR3, 1600 MHz, 0x80AD, 0x484D54343147533641465238412D50422020
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x112), Broadcom BCM43xx 1.0 (7.77.106.3 AirPortDriverBrcmNIC-1435.3)
Bluetooth: Version 7.0.1f1, 3 services, 27 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
Network Service: VEMEO, PPP (L2TP), ppp0
Serial ATA Device: APPLE SSD SM0512F, 500,28 GB
USB Device: USB 3.0 Bus
USB Device: Apple Internal Keyboard / Trackpad
USB Device: BRCM20702 Hub
USB Device: Bluetooth USB Host Controller
USB Device: USB Gaming Mouse
Thunderbolt Bus: MacBook Pro, Apple Inc., 17.2

```
## 👟 Steps to Reproduce (S2R)
Just start it in the new macOS

**Comment by sidneys** [comment]
@shoguevara 

Thanks for your report.

The problem cannot currently be reproduced, as the app testing environments have not been migrated to Catalina yet. Does the issue persist in the new release [v10.3.0](https://github.com/sidneys/pb-for-desktop/releases/tag/v10.3.0)?

Kind regards

**Comment by sidneys** [comment]
@shoguevara 

The issue was successfully reproduced and is due to an upstream bug related to the electron-builder packaging & notarization pipeline, which has not been solved completely yet (https://github.com/electron-userland/electron-builder/issues/4299). However, a hotfix for PB for Desktop is possible and will be released shortly, will full Catalina support.

Cheers, S

**Comment by sidneys** [comment]
@shoguevara 

This issue should have been resolved as of the brand new release [v10.5.0](https://github.com/sidneys/pb-for-desktop/releases).

Please review and update this issue accordingly.

Kind regards, S
